src/modal/modal.js

```diff
@@ -454,6 +456,7 @@ angular.module('ui.bootstrap.modal', [])
                 backdropClass: modalOptions.backdropClass,
                 windowClass: modalOptions.windowClass,
                 windowTemplateUrl: modalOptions.windowTemplateUrl,
+                container: $modalProvider.defaultContainer,
```

Comment:
Looking at the original issue this PR references, any particular reason for the choice of `container` over `appendTo`? I think the latter makes more sense semantically, but am open to other options.